[Issue] CI (Multi-Arch CI ASAN) (DEB/RPM): `build_package.py` fails with empty version string (`Version string '' does not have major.minor versions`)

## Summary:
The job fails during Linux packaging config generation due to an empty version string being passed into version parsing logic.

## Run Links
- https://github.com/ROCm/TheRock/actions/runs/25355434313/job/74384302618
- https://github.com/ROCm/TheRock/actions/runs/25355434313/job/74384302738

## Error
```
Run # GFX architectures are auto-detected from the artifact directory
Traceback (most recent call last):
  File "/home/runner/_work/TheRock/TheRock/./build_tools/packaging/linux/build_package.py", line 1107, in <module>
Auto-detected GFX architectures: ['gfx94X-dcgpu', 'gfx950-dcgpu']
    main(sys.argv[1:])
  File "/home/runner/_work/TheRock/TheRock/./build_tools/packaging/linux/build_package.py", line 1103, in main
    run(args)
  File "/home/runner/_work/TheRock/TheRock/./build_tools/packaging/linux/build_package.py", line 942, in run
    config = create_package_config(args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/_work/TheRock/TheRock/./build_tools/packaging/linux/build_package.py", line 906, in create_package_config
    raise ValueError(
ValueError: Version string '' does not have major.minor versions
Error: Process completed with exit code 1.
```

--- a/.github/workflows/multi_arch_ci_asan.yml
+++ b/.github/workflows/multi_arch_ci_asan.yml
@@ -58,6 +58,8 @@ jobs:
       build_config: ${{ needs.setup.outputs.linux_build_config }}
       test_labels: ${{ needs.setup.outputs.linux_test_labels }}
       rocm_package_version: ${{ needs.setup.outputs.rocm_package_version }}
+      rocm_deb_package_version: ${{ needs.setup.outputs.rocm_deb_package_version }}
+      rocm_rpm_package_version: ${{ needs.setup.outputs.rocm_rpm_package_version }}
       test_type: ${{ needs.setup.outputs.test_type }}
     permissions:
       contents: read

--- a/.github/workflows/multi_arch_ci_asan.yml
+++ b/.github/workflows/multi_arch_ci_asan.yml
@@ -43,6 +43,9 @@ jobs:
     with:
       build_variant: "asan"
       linux_amdgpu_families: ${{ inputs.linux_amdgpu_families || '' }}
+      linux_test_labels: ${{ inputs.linux_test_labels || '' }}
+      prebuilt_stages: ${{ inputs.prebuilt_stages || '' }}
+      baseline_run_id: ${{ inputs.baseline_run_id || '' }}
 
   linux_build_and_test:
     name: Linux::${{ fromJSON(needs.setup.outputs.linux_build_config || '{}').build_variant_label || 'skip' }}

